views-fieldset/src/main/java/io/micronaut/views/fields/Option.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.views.fields;
+
+import io.micronaut.core.annotation.NonNull;
+
+import java.util.Objects;
+
+/**
+ * HTML Option.
+ * <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option">Option</a>
+ * @author Sergio del Amo
+ * @since 4.1.0
+ */
+public class Option implements FormElement {
```

Comment:
Should this have `@Introspected(builder = @Introspected.IntrospectionBuilder(builderClass = Option.Builder.class))`?

Reply:
And if it should, do we need a [test in OptionTest](https://github.com/micronaut-projects/micronaut-views/pull/613/files#diff-7c5cb0b94d00a2e57e54a03e2c26294b652c348f8c69bd1584bf5ab717dce972) as we do with RadioTest for example